README.md

```diff
@@ -1,2 +1,53 @@
-# public
-Public Repository of Social Income
+<h1 align="center">
```

Comment:
nit: I think it's cleaner to not use html tags in markdown. I would replace it with proper markdown as you do in the sections afterwards